pkg/controller/datavolume-controller.go

```diff
@@ -439,6 +443,21 @@ func (r *DatavolumeReconciler) getSnapshotClassForSmartClone(dataVolume *cdiv1.D
 	return "", errors.New("could not match snapshotter with storage class, falling back to host assisted clone")
 }
 
+func (r *DatavolumeReconciler) getCloneStrategy() (cdiv1.CDICloneStrategy, error) {
+	crList := &cdiv1.CDIList{}
```

Comment:
All of this code is duplicated from pkg/controller/util.go from the GetWorkloadNodePlacement function. Can we make function in util called: GetCDICR? Something like:
```go
func GetCDICR(c client.Client) (*cdiv1.CDI, error) {
	crList := &cdiv1.CDIList{}
	if err := c.List(context.TODO(), crList, &client.ListOptions{}); err != nil {
		return nil, err
	}
	if len(crList.Items) != 1 {
		return nil, fmt.Errorf("Number of CDI CRs != 1")
	}
	return &crList.Items[0], nil
}
```
And then calling that from all places we need to get the CDI CR?